Documentation/user-manual.txt

```diff
@@ -19,7 +19,7 @@ image::images/Subsurface4Banner.jpg["Banner",align="center"]
 Linus Torvalds, Miika Turkia, Amit Chaudhuri, Jan Schubert, Salvador Cuñat, Pedro Neves,
 Stefan Fuchs
 
-[blue]#_Version 4.8, September 2018_#
+[blue]#_Version 4.8.4, January 2019_#
```

Comment:
Should this be 4.9 (as that's going to be the next version where these changes apply?)
And then it should also be February, right?

Reply:
done